app/models/book.rb

```diff
@@ -88,6 +88,75 @@ class Book < ActiveRecord::Base
   has_and_belongs_to_many :recommendations
 
   CSV_COLUMNS = ["Book Name", "ASIN"]
+  LEVELS_DICT = {
```

Comment:
Basically Worldreader currently has all their books labeled as level "A", "B", and so on. However, there are classifications of levels, i.e. "Primary 1" and so on that correspond to one or more of those types of levels. Each one is stored in a "tag" with "tagtype" Levels (and its own level), so I'm making a dictionary to change the higher level classifications to lower level ones.